src/main/java/com/blog/web/security/SecurityUtil.java

```diff
@@ -0,0 +1,16 @@
+package com.blog.web.security;
+
+import org.springframework.security.authentication.AnonymousAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+public class SecurityUtil {
+    public static String getSessionUser() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        if(!(authentication instanceof AnonymousAuthenticationToken)) {
```

Comment:
return can be made to a ternary ->
!(authentication instanceof AnonymousAuthenticationToken) ? authentication.getName():null